app/assets/javascripts/initializers/runtime.js

```diff
@@ -133,4 +133,19 @@ class Runtime {
       }
     });
   }
+
+  /**
+   * Returns true if the supplied KeyboardEvent includes the OS-specific
+   * modifier key. For example, the Cmd key on Apple platforms or the Ctrl key
+   * on others.
+   *
+   * @param {KeyboardEvent} The event to check for the OS-specific modifier key
+   *
+   * @returns {Boolean} true if the event was fired with the OS-specific
+   *                    modifier key, false otherwise. Also returns false if
+   *                    the event is not a KeyboardEvent.
+   */
+  static hasOSSpecificModifier(event) {
+    return navigator.userAgent.indexOf('Mac OS X') != -1;
+  }
```

Comment:
We're missing the check that looks for the ctrl or cmd key - I think it just got lost in the refactor moving it into the helper class here 🙂 

At the minute if I just press e.g. the `i` key on my mac I always get italics, even without pressing `cmd`

Reply:
hahaha Oops! 🤦🏻‍♂️

Reply:
![I'll just warn you now, I don't know how to use a computer](https://user-images.githubusercontent.com/108205/128748486-4d7d8a7c-fa89-4e31-8096-8f516915f8e8.jpeg)

Reply:
we've all been there 😂